src/patients/patients_search.tsx

```diff
@@ -1,5 +1,5 @@
-import { FunctionComponent, useState } from "react";
-import { Patient, PatientSearchQuery } from "./patients";
+import { FunctionComponent, useState } from 'react';
+import { Patient, PatientSearchQuery } from './patients';
```

Comment:
This seems to be formatted this way only in this file. Could you apply the same format to old the files?